src/test/kotlin/PlusTimeTest.kt

```diff
@@ -8,55 +8,55 @@ import kotlin.test.assertFailsWith
 internal class PlusTimeTest {
     @Test
     fun `plus time in hours`() {
-        val firstDate = "2013-08-11T17:22:04.51+01:00"
```

Comment:
How come you changed this? This input should be allowed and work.

Reply:
the issue is not in the input , it is accepted. Joda-Time was not that accurate in this case. if you pass milli-seconds then the output also should be assessed including milii-seconds , I adjusted the tests now to reflect that

Reply:
I had this in there resultDate.toString(ISODateTimeFormat.dateTimeNoMillis(), so dateTimeNoMillis() was doing the trick before. Input can be anything, just the output should not contain millis

Reply:
That's the reason the tests were set up like this.

Reply:
Fixed!